src/DataDepsGenerators.jl

```diff
@@ -87,6 +87,48 @@ include("DataCite.jl")
 include("Figshare.jl")
 include("JSONLD/JSONLD.jl")
 
+function aggregate(generator_meta)
+
+    meta = Metadata(
+        combine_all(generator_meta, :shortname),
+        combine_all(generator_meta, :fullname),
+        combine_all(generator_meta, :website),
+        combine_all(generator_meta, :description),
+        combine_all(generator_meta, :author),
+        combine_all(generator_meta, :maintainer),
+        combine_all(generator_meta, :license),
+        combine_all(generator_meta, :published_date),
+        combine_all(generator_meta, :create_date),
+        combine_all(generator_meta, :modified_date),
+        combine_all(generator_meta, :paper_cite),
+        combine_all(generator_meta, :dataset_cite),
+        combine_all(generator_meta, :dataurls),
+        combine_all(generator_meta, :datachecksums),
+    )
+    body(meta)  
+end
+
+function getfieldlist(generator_meta::Vector, sym::Symbol)
+    return [getfield(i, sym) for i in generator_meta]
+end
+
+function combine_all(values::Vector, sym::Symbol)
+    ret = missing
+    for value in getfieldlist(values, sym)
+        ret = combine(ret, value, sym)
+    end
+    ret
+end
+
```

Comment:
You don't quiet have dispatch on value working correctly.
Talk to me, or someone on slack.

Reply:
See:

https://stackoverflow.com/questions/51568006/how-do-i-do-dispatch-on-value-in-julia#51568007

Reply:
Solved now